CIP-0062/README.md

```diff
@@ -0,0 +1,473 @@
+might be a subset of Catalyst proposals, a council election, or even some
+private purpose (agreed by convention). Currently, only voting purpose 0 (Zero)
+is defined, and it is for Catalyst events.
+
+**IMPORTANT**: The List of purposes in this CIP should be considered the
+authoritative list of known purposes, subject to future amendment. Other voting
+purposes will be defined as required, by either an update to this CIP or a
+future CIP listing currently allocated Voting Purposes.
+
+### VotingCredentials
+```ts
+interface VotingCredentials {
+  votingKey: PublicKey
+  stakingCredential: PublicKey
+}
+```
+Information used to represent a wallet's voting credentials.
+
+* votingKey - Derivation as described within [CIP-0036](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036), wallets should use `address_index`= 0.
+* stakingCredential - At the moment, the only supported staking credential is a public staking key. This is used to represent the wallet's staked ADA.
+
+### BlockDate
+
+```ts
+interface BlockDate {
+  epoch: number
+  slot: number
+}
+```
+
+* epoch - an epoch value on the voting block chain.
+* slot - a block slot number on the voting block chain.
+
+#### Proposal
+
+```ts
+interface Proposal {
+  votePlanId: string;
+  proposalIndex: number;
+  voteOptions?: number[];
+  voteEncKey?: string;
+}
+```
+
+Proposal information.
+
+* votePlanId - Hex encoded string to represent vote plan unique identifier. This will be the same as anchored on voting chain, and is managed by the dApp.
+* proposalIndex  - The index of the proposal within the vote plan.
+* voteOptions - *Optional*. Total number of vote options.  Only present in a private/encrypted vote.
+* voteEncKey - *Optional*. If this vote is Private, this field is present and contains the key used to encrypt the vote.  When the vote is public, this key is absent from the proposal.
+
+### Vote
+
+```ts
+interface Vote {
+  proposal: Proposal;
+  choice: number;
+  expiration: BlockDate;
+  purpose: VotingPurpose;
+  spendingCounter?: number;
+  spendingCounterLane?: number;
+}
+```
+
+An individual raw Unsigned Vote record.
+
+* proposal - The [proposal](#proposal) being voted on.
+* choice - This value MUST match one of the available `voteOptions` in the `proposal` element. An `UnknownChoiceError` should be thrown if the value is not one of the valid `voteOptions` of the `proposal`.
+* expiration - Voting Chain epoch \& slot for when the vote will expire. This value is supplied and maintained by the dApp, and forms a necessary component of a [Vote](#vote)
+* purpose - The [voting purpose](#voting-purpose) being voted on. (Currently always 0).
+* spendingCounter - Optional, but if present, the spending counter is used to prevent double voting. The spending counter for the vote transaction will be supplied and maintained by the dApp. The dApp will manage supplying this in the correct order, and this should not be enforced by the Wallet.
```

Comment:
if not provided, what value, if any should this default to?

Reply:
These fields should default to 0 - thanks for raising this I will make it explicit in the specification.